app/models/order_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Order, type: :model do
+  it { should belong_to(:customer) }
+  
+  it { should validate_presence_of(:product_name) }
+  it { should validate_presence_of(:product_count) }
+  it { should validate_presence_of(:customer) }
+end
```

Comment:
Heads up that all specs belong in the `specs` directory. It looks like you have a spec file at `spec/models/order_spec.rb` so you can delete this one.